Cargo.toml

```diff
@@ -11,12 +11,12 @@ description = "Smart contract for AZERO coin wrapped as a PSP22 token"
 exclude = [ ".github/*" ]
 
 [dependencies]
-ink = { version = "4.3", default-features = false }
+ink = { version = "4.3.0", default-features = false }
```

Comment:
Let's fix for `=4.3.0`.

Reply:
Doesn't matter: there won't be any more ink 4.x.x releases, and on top of that this is "build-once-and-never-again" project

Reply:
> Doesn't matter: there won't be any more ink 4.x.x releases

It's assumption.

Reply:
Done